.scripts/util.sh

```diff
@@ -63,52 +64,49 @@ is_generated() {
     fi
 }
 
-select_browser() {
+select_browser_caller() {
     local browser_engine="$1"
+    MSWEBVIEW_INCLUDE_DIR=$PACKAGE_DIR/Microsoft.Web.WebView2.$MSWEBVIEW_VERSION/build/native/include
+    MSWEBVIEW_LIB_DIR=$PACKAGE_DIR/Microsoft.Web.WebView2.$MSWEBVIEW_VERSION/build/native/x64
+    CHROMIUM_INCLUDE_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/include
+    CHROMIUM_LIB_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/x64
 
     if [ "$browser_engine" = "Chromium" ]; then
         BROWSER="Chromium"
         BROWSER_INCLUDE_DIR="$CHROMIUM_INCLUDE_DIR"
         BROWSER_LIB_DIR="$CHROMIUM_LIB_DIR"
-        SYS_OPTS="-DCHROMIUM_INCLUDE_DIR=$BROWSER_INCLUDE_DIR -DCHROMIUM_LIB_DIR=$BROWSER_LIB_DIR"
+        SYS_OPTS="-DBROWSER_INCLUDE_DIR=$BROWSER_INCLUDE_DIR -DBROWSER_LIB_DIR=$BROWSER_LIB_DIR -DBROWSER=$BROWSER"
     elif [ "$browser_engine" = "MSWebView2" ]; then
         BROWSER="MSWebView2"
         BROWSER_INCLUDE_DIR="$MSWEBVIEW_INCLUDE_DIR"
         BROWSER_LIB_DIR="$MSWEBVIEW_LIB_DIR"
-        SYS_OPTS="-DMSWEBVIEW_INCLUDE_DIR=$BROWSER_INCLUDE_DIR -DMSWEBVIEW_LIB_DIR=$BROWSER_LIB_DIR"
-    else
-        throw_error "Unsupported browser engine option: $browser_engine. Program will terminate."
-        return 1
+        SYS_OPTS="-DBROWSER_INCLUDE_DIR=$BROWSER_INCLUDE_DIR -DBROWSER_LIB_DIR=$BROWSER_LIB_DIR -DBROWSER=$BROWSER"
     fi
```

Comment:
Maybe add a final catch conditional incase the user has mis-typed, wherein:

Set `chromium` as the default `BROWSER` and print a message to let the user know that this has happened:
    - "$1 was not a valid option from $BROWSER_OPTIONS. 'chromium' was automatically selected."